README.md

```diff
@@ -239,6 +239,24 @@ LanguageTheme object:
 
 A set of example shapes can be found [here](https://shexshapes.inrupt.net/public/), which show various ShEx shapes.
 
+### ShexForm, FormModel
+See https://github.com/inrupt/solid-react-components#shexformbuilder
+
+### LiveUpdate, useLiveUpdate
+See https://github.com/solid/react-components#automatically-refresh-when-data-is-updated
+
+### useWebId
+See https://github.com/solid/react-components#identify-the-user
+
+### Image
+See https://github.com/solid/react-components#load-data-from-the-user-and-from-the-web
+
+### UpdateContext
+See https://github.com/solid/react-components/blob/master/src/UpdateContext.js
+
+### AutoSaveDefaultSpinner
+A spinner for letting the user know when a form field was auto-saved.
```

Comment:
This is the only addition that might need some additional explanation. The AutoSaveDefaultSpinner is simply an example of how to make a default spinner, and is only exported so we can test it in App.js. 

We even have a completely different spinner [in the generator](https://github.com/inrupt/generator-solid-react/blob/develop/generators/app/templates/src/components/AutoSaveSpinner/index.js) and don't use this one.  In fact, this one would be a very poor one to use.

Reply:
OK so then please remove this poor one and add the spinner you recommend! :)